modules/renter/repair.go

```diff
@@ -126,7 +126,11 @@ func (r *Renter) addFileToRepairState(rs *repairState, file *file) {
 		// Check whether this contract is offline. Even if the contract is
 		// offline, we want to record that the chunk has attempted to use this
 		// contract.
-		offline := r.hostContractor.IsOffline(contract.ID)
+		var offline bool
+		renterContract, exists := r.hostContractor.Contract(contract.ID)
+		if exists && renterContract.InGoodStanding {
+			offline = true
+		}
```

Comment:
going to keep it named offline, want to disturb this code as little as possible. It'll be given a facelift soon enough. Can do `offline := !exists || !InGodStanding` though.